evm_arithmetization/src/cpu/kernel/asm/util/assertions.asm

```diff
@@ -56,10 +56,8 @@ global panic:
 %endmacro
 
 %macro assert_gt
-    // %assert_zero is cheaper than %assert_nonzero, so we will leverage the
-    // fact that (x > y) == !(x <= y).
-    LE
-    %assert_zero
+    GT
+    %assert_nonzero
```

Comment:
nit: would you mind updating also the `%macro assert_lt(ret)` above as `GE` doesn't exist either?